torch/csrc/jit/codegen/cuda/manager.cpp

```diff
@@ -110,6 +129,30 @@ class CudaFusionManager {
     return graph_cache_[kernel_id]->runGraphWithInputs(inputs);
   }
 
+  bool hasFallbackCode(int32_t kernel_id) {
+    std::lock_guard<std::mutex> guard(mutex_);
+    return graph_cache_.count(kernel_id);
```

Comment:
shouldn't this be `fallback_cache_` instead?

Reply:
Whoops, thanks for catching that!